### ISSUE TITLE
DocumentDb Emulator can not use TCP Connection Protocol

### ISSUE THREAD

**Issue by aurimielisss** [Azure/azure-documentdb-dotnet#192]
hey, I am having issues with ports here.

var connectionPolicy = new ConnectionPolicy
            {
                ConnectionMode = ConnectionMode.Direct,
                ConnectionProtocol = Protocol.Tcp
            };

seems not working on local environment. maybe port configuration is bad.
"EndpointUri": "https://localhost:8081"

gives me error: ServiceUnavailableException: Service is currently unavailable.

is that true?

**Comment by rnagpal** [comment]
@aurimielisss In order to debug this, we need to collect debugging traces. Please ensure that you have downloaded the latest version of the DocumentDB emulator.

Can you run the following from an admin command prompt:

cd /d "%ProgramFiles%\DocumentDB Emulator"
DocumentDB.Emulator.exe /starttraces
DocumentDB.Emulator.exe
Perform the scenario
DocumentDB.Emulator.exe /stoptraces
You should find an docdbemulator_000001.etl file in the Program Files\DocumentDB Emulator.  Please send that along for debugging.

**Comment by rnagpal** [comment]
@aurimielisss Following up on this error you were seeing using the emulator? Is this resolved now? If not please send us the trace files and we can investigate it further.

**Comment by rnagpal** [comment]
@aurimielisss Just following up on this. Does Gateway mode works for you? Are you using .NET Core SDK(Microsoft.Azure.DocumentDB.Core) or the .NET Framework SDK(Microsoft.Azure.DocumentDB)?

**Comment by aurimielisss** [comment]
hi, let me try again to replicate this issue, I am using Microsoft.Azure.DocumentDB.Core.

**Comment by rnagpal** [comment]
@aurimielisss In that case it's a dup of https://github.com/Azure/azure-documentdb-dotnet/issues/194 and is not related to emulator. This will fail for a production endpoint as well. I'll close this issue and you can get the status here: https://github.com/Azure/azure-documentdb-dotnet/issues/194

**Comment by aurimielisss** [comment]
ok, thanks
